src/test/java/it/com/gradle/develocity/bamboo/injection/MavenInjectionTest.java

```diff
@@ -155,4 +156,31 @@ void buildScanNotPublishedWithoutAcceptingTos() {
         assertThat(output, containsString("[INFO] The build scan was not published due to a configuration problem."));
         assertThat(output, containsString("[INFO] The Gradle Terms of Service have not been agreed to."));
     }
+    @Test
+    void extensionAlreadyAppliedInProjectAndBuildScanAttemptedToPublishToProjectConfiguredHost() {
+        // given
+        ensurePluginConfiguration(form -> form
+                .setServer(PUBLIC_DEVELOCITY_SERVER)
+                .enableGeExtensionAutoInjection()
+        );
+
+        PlanKey planKey = PlanKeys.getPlanKey(PROJECT_KEY, "MPEA");
```

Comment:
I guess it's a project with the DV extension already configured?

Reply:
Exactly, but I haven't updated the test resources yet, so when testing locally it's still absent

Reply:
Updated now